sklearn/utils/tests/test_tags.py

```diff
@@ -43,8 +37,9 @@ class EmptyRegressor(RegressorMixin, BaseEstimator):
     pass
 
 
+# TODO(1.8): Update when implementing __sklearn_tags__ is required
 @pytest.mark.filterwarnings(
-    "ignore:.*no __sklearn_tags__ attribute.*:DeprecationWarning"
+    "ignore:.*no attribute '__sklearn_tags__'.*:DeprecationWarning"
 )
 @pytest.mark.parametrize(
     "estimator, value",
```

Comment:
This filter used to not filter anything but didn't have to :)
Now that I've made it so that a warning is raised for estimator not implementing `__sklearn_tags__`, it triggers and showed that the filter was wrong.